chapters/research-design.tex

```diff
@@ -1,288 +1,593 @@
-Instead, we make inferences from samples:
-we try to devise a comparison group that evidence suggests
-would be identical to the treated group had they not been treated.
-
-This \textbf{control group} serves as a counterfactual to the treatment group,
-\index{control group}\index{treatment group}
-and we compare the distributions of outcomes within each
-to make a computation of how different the groups are from each other.
+%-----------------------------------------------------------------------------------------------
+%-----------------------------------------------------------------------------------------------
+
+\section{Causality, inference, and identification}
+
+The primary goal of research design is to establish \textbf{identification}
+for a parameter of interest -- that is, to demonstrate
+a source of variation in a particular input that has no other possible channel
+to alter a particular outcome, in order to assert that some change in that outcome
+was caused by that change in the input.
+  \index{identification}
+When we are discussing the types of inputs commonly referred to as
+``programs'' or ``interventions'', we are typically attempting to obtain estimates
+of a program-specific \textbf{treatment effect}, or the change in outcomes
+directly attributable to exposure to what we call the \textbf{treatment}.\cite{abadie2018econometric}
+  \index{treatment effect}
+When identification is believed, then we can say with confidence
+that our estimate of the treatment effect would,
+with an infinite amount of data,
+give us a precise estimate of that treatment effect.
+Under this condition, we can proceed to draw evidence from the limited samples we have access to,
+using statistical techniques to express the uncertainty of not having infinite data.
+Without identification, we cannot say that the estimate would be accurate,
+even with unlimited data, and therefore cannot associate it to the treatment
+in the small samples that we typically have access to.
+Conversely, more data is not a substitute for a well-identified experimental design.
+Therefore it is important to understand how exactly your study
+identifies its estimate of treatment effects,
+so you can calculate and interpret those estimates appropriately.
+All the study designs we discuss here use the \textbf{potential outcomes} framework
+to compare a group that received some treatment to another, counterfactual group.
+Each of these types of approaches can be used in two contexts:
+\textbf{experimental} designs, in which the research team
+is directly responsible for creating the variation in treatment,
+and \textbf{quasi-experimental} designs, in which the team
+identifies a ``natural'' source of variation and uses it for identification.
+Neither type of approach is implicitly better or worse,
+and both are capable of achieving effect identification under different contexts.
```

Comment:
"experimental" and "quasi"

Reply:
ok, I think I was confused because in line 67 "approaches" refers to the different methods in the chapter, and in line 72, "type of approach" it refers to "contexts", except that "contexts" is used again in line 77 meaning something else.

How about
- "each of these designs can be used in two settings" in line 67
- "neither setting" in line 76

Reply:
fa922de7df148f33f51d430a250f2f57c2fac5ec